### DISCUSSION TITLE
API to Check Server Status

### DISCUSSION THREAD

**Discussion by Tailslide** [pymedusa/Medusa#10152] [Ideas & Feature requests]
I would like to be able to restart my pymedusa docker container if the post process task has degraded to an 'alive = false' status on the status screen. I don't see anywhere in the API docs to check for this.  Is there some way to check this?

Edit: I have a script to check this I guess this isn't the correct place for it.. if anyone needs it you can DM me.

**Comment by OmgImAlexis** [comment]
We need a `/api/v2/healthz` which should return a 200 when all is good and another when it's not.

**Comment by Tailslide** [comment]


**Reply by OmgImAlexis** [reply]
Please use reply comments instead of starting a new top comment for each.
